pkg/client/results/raw.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright the Sonobuoy contributors 2019
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package results
+
+import (
+	"os"
+)
+
+// processRaw will return an Item object with the File value set to the path in question. If the
+// file is unable to be stat'd then the status of the Item is StatusFailed (StatusPassed otherwise).
+func processRawFile(rawPath string) (Item, error) {
+	status := StatusPassed
+	if _, err := os.Stat(rawPath); err != nil {
+		status = StatusFailed
+	}
+
+	return Item{
+		Name:   rawPath,
```

Comment:
Make this `filepath.Base` of the path. Even though it would be duplicate info, may want to add the `metadata.file` with the full path to be consistent with how the junit results set that field.